boards.txt

```diff
@@ -25,6 +27,8 @@ uno2018.build.extra_flags={build.328emulation} -DMILLIS_USE_TIMERB3
 #uno2018.build.extra_flags=-B{runtime.tools.atpack.path}/gcc/dev/{build.mcu}
 
 uno2018.bootloader.tool=avrdude
+uno2018.bootloader.tool.serial=avrdude
+uno2018.bootloader.tool.network=arduino_ota
```

Comment:
I'm quite sure we don't want to update the bootloader via OTA :) Should we drop this line to explicitly provoke an error?

Reply:
I don't know really. Arduino CLI technically supports it but I have no idea if it's feasible or not.

Reply:
@cmaglie

Reply:
No, it's not possible because we don't have a `tools.arduino_ota.bootloader.pattern` in platform.txt